README.md

```diff
@@ -10,7 +10,22 @@ First, you'll need to configure a database. Once this project is deployed, we'll
 
 ### Windows setup
 
-Coming Soon!
+## Chocolatey and Postgres (Needs Testing)
+1. [Install Choclatey](https://chocolatey.org/docs/installation#installing-chocolatey) if you do not already have it installed.
+2. Add `C:\ProgramData\chocolatey\bin` to your paths (type path in search, click Edit environment varaibles for you account).
+3. In your cmd.exe (admin) terminal run `choco install postgresql --params '/Password:<assigned to postgres user>` (cmd.exe (admin) can be found by pressing `win + x`).
```

Comment:
I think this is missing a closing single quote.

Reply:
If we are able to explicitly set a username, I'd prefer that params. i'll look at the documentation when it's back up. 
I'd also like to include a full example of the command that requires no user guessing what the `<assigned to postgres user>` means.

Reply:
Ok.  That is from the Chocolatey docs and params password is assigned to pastgres before you create a user.  Maybe better wording would be: <choose a password>. Your password will be assigned to the default postgres superuser.

Reply:
I removed the --param all together after reading what Aaron said about it prompting for a password anyways.